estuary/utils/story.py

```diff
@@ -0,0 +1,468 @@
+            }
+        else:
+            raise ValueError('The label should belong to a Neo4j node class')
+
+    def is_valid(self, story):
+        """
+        Determine if the story path matches the returned story.
+
+        :param str label: Neo4j node label
+        :return: whether story is valid for this story path
+        :rtype: bool
+        """
+        # This is the default story flow if no other story flow works out
+        return True
+
+
+class ModuleStoryManager(ContainerStoryManager):
+    """A class containing utility methods to create a module story."""
+
+    def __init__(self):
+        """Instantiate variables required to get module story of a node."""
+        self.story_flow_list = ['BugzillaBug', 'DistGitCommit', 'KojiBuild',
+                                'ModuleKojiBuild', 'Advisory', 'FreshmakerEvent',
+                                'ContainerKojiBuild', 'ContainerAdvisory']
+
+    def story_flow(self, label):
+        """
+        Get the next/previous node in a story flow/pipeline path.
+
+        :param str label: Neo4j node label
+        :return: uid and relationship information in both forward and backward directions
+        :rtype: dict
+        """
+        if label == ModuleKojiBuild.__label__:
+            return {
+                'uid_name': ModuleKojiBuild.id_.db_property or ModuleKojiBuild.id.name,
+                'forward_relationship': '{0}<'.format(
+                    KojiBuild.advisories.definition['relation_type']),
+                'forward_relationship_display': 'that contain',
+                'forward_label': Advisory.__label__,
+                'forward_label_display': 'advisory',
+                'backward_relationship': '{0}>'.format(
+                    ModuleKojiBuild.components.definition['relation_type']),
+                'backward_relationship_display': 'that contain',
+                'backward_label': KojiBuild.__label__,
+                'backward_label_display': 'build'
+            }
+
+        flow_dict = super(ModuleStoryManager, self).story_flow(label)
+
+        if label == KojiBuild.__label__:
+            flow_dict.update({
+                'forward_relationship': '{0}<'.format(
+                    KojiBuild.module_builds.definition['relation_type']),
+                'forward_relationship_display': 'attached to',
+                'forward_label': ModuleKojiBuild.__label__,
+                'forward_label_display': 'module build'
+            })
+        elif label == Advisory.__label__:
+            flow_dict.update({
+                'backward_label': ModuleKojiBuild.__label__,
+                'backward_label_display': 'module build'
+            })
+
+    def is_valid(self, story):
+        """
+        Determine if the story path matches the returned story.
+
+        :param str label: Neo4j node label
+        :return: whether story is valid for this story path
+        :rtype: bool
+        """
+        for model in [node[0][0].nodes for node in story]:
```

Comment:
Can you assign this to a variable:
```python
nodes = [node[0][0].nodes for node in story]
for node in nodes:
    if node.__label__ == ModuleKojiBuild.__label__:
        return True
return False
```

Reply:
I feel it's just creating a useless variable which is used only once.